jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java

```diff
@@ -189,7 +190,8 @@ static Instant getCreationTime(String imageReference) throws IOException, Interr
     return Instant.parse(inspect);
   }
 
-  static void assertImageDigestAndId(Path projectRoot) throws IOException, DigestException {
+  static void assertValidImageDigestAndIdOutputFiles(Path projectRoot)
```

Comment:
hm how about `assertThatExpectedImageDigestAndIdReturned`?

Reply:
Fixed in #3179.